src/sentry/web/frontend/organization_settings.py

```diff
@@ -57,17 +90,32 @@ def get_form(self, request, organization):
                 'allow_joinleave': bool(organization.flags.allow_joinleave),
                 'enhanced_privacy': bool(organization.flags.enhanced_privacy),
                 'allow_shared_issues': bool(not organization.flags.disable_shared_issues),
+                'scrub_data': bool(organization.get_option('sentry:scrub_data', False)),
```

Comment:
We need them to simply mean something different.

From an org level you are selecting one of those two options, so the option doesn't simply mean "enable or disable it". Its more "if I've said its enabled, you cant disable it". We could also support the "Disabled for all projects" but in all honesty that's not super useful.